src/app/BE/card/templates/Card.js

```diff
@@ -35,7 +35,27 @@ const DefaultCard = ({ parentCode, actions = [], code, columns }) => {
   const userCode = useSelector(selectCode('USER'), equals)
   const userType = getUserType(useSelector(selectCode(userCode), sameLength))
 
-  return (
+  const mentorName = useSelector(selectCode(code, '_LNK_MENTOR__PRI_NAME'), sameValue)
+  const mentorImage = useSelector(
+    selectCode(code, '_LNK_MENTOR__PRI_USER_PROFILE_PICTURE'),
+    sameValue,
+  )
+  const mentorIndustry = useSelector(
+    selectCode(code, '_LNK_MENTOR__LNK_MM_INDUSTRY__PRI_NAME'),
+    sameValue,
+  )
+
+  const menteeName = useSelector(selectCode(code, '_LNK_MENTEE__PRI_NAME'), sameValue)
+  const menteeImage = useSelector(
+    selectCode(code, '_LNK_MENTEE__PRI_USER_PROFILE_PICTURE'),
+    sameValue,
+  )
+  const menteeIndustry = useSelector(
+    selectCode(code, '_LNK_MENTEE__LNK_MM_INDUSTRY__PRI_NAME'),
+    sameValue,
+  )
+
+  return realm !== 'mentormatch' ? (
```

Comment:
Please don't do this in any part of the application ever, we are trying to move away from conditional logic based on the realm. Probably create a new file instead of trying to make it generic and using realm as the conditional chain.